cmd/lift/lift.go

```diff
@@ -106,6 +108,15 @@ func lift(chainFile string, inFile string, outFile string, faFile string, unMapp
 	}
 }
 
+// QuerySeq takes in a slice of Fasta and a position (name and index) and returns true if a query
```

Comment:
This was a function is the fasta package, but it seemed to have a rather specific use case so I moved it to the cmd that calls it. That said my conviction for the move is mild at best so if people thinks this belongs in fasta then I will move it back. Otherwise I will leave the comment change to Riley.